lib/preload.js

```diff
@@ -17,10 +17,23 @@
   };
 
   window.onerror = function(msg, file, line, column, err){
-    if (err && msg.indexOf(err.stack) > -1) {
-      err.stack = err.stack + '\n  at ' + file + ':' + line + ':' + column
+    if (err) {
+      console.error((err.stack || err.message || err) + '\n  at ' + file + ':' + line + ':' + column);
+    } else if (msg instanceof Event) {
+      console.error('Unhandled error event at', msg.target, '"' + msg.target + '"')
+    } else {
+      var out = msg;
```

Comment:
Do you know in which case this condition would happen?

Reply:
Tbh. no, i added it for "in case". Since it seems to be a possibility, from the argument pattern.

Reply:
Could you please add a comment for this, as it wasn't obvious to me?